examples/mro_hirise_isis_cmp.ipynb

```diff
@@ -0,0 +1,394 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Comparing a USGSCSM and ISIS camera for Dejittered HiRISE image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyproj\n",
+    "import pvl\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import json\n",
+    "import knoten\n",
+    "import csmapi\n",
+    "\n",
+    "os.environ['ISISROOT'] = '/usgs/pkgs/isis3.8.0_RC1/install'\n",
+    "from pysis import isis\n",
+    "from pysis.exceptions import ProcessError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Update the ISIS cube\n",
+    "Requires ESP_016076_2175_REDmos_hijitreged.balance.noproj.8bit.cub in data directory\n",
+    "\n",
+    "We also have to set the shapemodel to the ellipsoid because CSM does not natively support DEM intersections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Failed to set shapemodel for data/ESP_016076_2175_REDmos_hijitreged.balance.noproj.8bit.cub to ellipsoid:\n",
```

Comment:
Should the output be showing that the editlab call failed?

Reply:
nope! I'll fix that